multi_import/formats.py

```diff
@@ -91,18 +92,8 @@ def __init__(self):
             _csv, 'application/csv', read_file_as_string=True
         )
 
-    @classmethod
-    def ensure_unicode(cls, file_contents):
-        charset = chardet.detect(file_contents)
-        encoding = charset['encoding']
-        encoding_confidence = charset['confidence']
-        if encoding and encoding_confidence > 0.5:
-            return file_contents.decode(encoding.lower()).encode('utf8')
-        else:
-            raise InvalidFileError(_(u'Unknown file type.'))
-
     def pre_read(self, file_object):
-        file_object = self.ensure_unicode(file_object)
+        file_object = six.ensure_text(file_object)
```

Comment:
@sibartlett `six.ensure_text(s)` does coercing the given string to `six.text_type` and I am wondering if it'd be good to use `ensure_text` in place of our `ensure_unicode` as this is shorter and also python 2 and 3 compatible. 
(Ref: https://six.readthedocs.io/#six.ensure_text)

Reply:
Spoke with @sibartlett and we keep our `ensure_unicode()` instead of using `six.ensure_text(s)`.